scripts/bootstrap.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -z "$IN_NIX_SHELL" ]; then
+  echo >&2 "Please, run this script from inside nix-shell."
+  exit 1
+fi
+
+deployment_repo="https://github.com/DisciplinaOU/disciplina-nixops"
+nixops_home="/var/lib/nixops"
+dname="dscp"
```

Comment:
It think it's cleaner to do `export NIXOPS_DEPLOYMENT=dscp` rather than passing it explicitly at each command.

Reply:
Also, the name should be `deployer` rather than `dscp`